src/app/helpers/error.interceptor.ts

```diff
@@ -1,21 +1,30 @@
 import {HttpEvent, HttpHandler, HttpInterceptor, HttpRequest} from '@angular/common/http';
 import {Injectable} from '@angular/core';
-import {Observable, throwError} from 'rxjs';
-import {LoginService} from '../services/login/login.service';
+import {Observable, throwError, Subscription} from 'rxjs';
+import {LoginService} from 'src/app/services/login/login.service';
 import {catchError} from 'rxjs/operators';
+import { Router } from '@angular/router';
+import { LocalStorageService } from '../services/local-storage.service';
```

Comment:
Nice work @patrickf949 , we can use `absolute imports` here

Reply:
@Tevinthuku done